site/create.md

```diff
@@ -19,8 +19,7 @@ The Create Domain Tool understands three domain types: `WLS`, `RestrictedJRF`, a
 
 To have the Create Domain Tool run RCU, simply add the `-run_rcu` argument to the previous command line and the RCU schemas will be automatically created.  Be aware that when the tool runs RCU, it will automatically drop any conflicting schemas that already exist with the same RCU prefix prior to creating the new schemas!
 
-It is also possible to specify the connection information in the model instead of 
-using the command line.  This is especially easier for complex jdbc connection string, such as RAC or Oracle Autonomous Transaction Processing Cloud Service databases connection string.  For information on how to use it, refer to [Specifying RCU connection information in the model](rcuinfo.md)
+It is also possible to specify the connection information in the model instead of using the command line arguments.  This is especially easier for database that requires complex jdbc connection string and parameters, such as RAC or Oracle Autonomous Transaction Processing Cloud Service database.  For information on how to use it, refer to [Specifying RCU connection information in the model](rcuinfo.md)
```

Comment:
the command line arguments -> the command-line arguments 
for database that requires complex jdbc connection string -> for a database that requires a complex JDBC connection string